safe_transaction_service/safe_messages/models.py

```diff
@@ -0,0 +1,68 @@
+from logging import getLogger
+
+from django.db import models
+from django.db.models import JSONField
+
+from hexbytes import HexBytes
+from model_utils.models import TimeStampedModel
+
+from gnosis.eth.django.models import EthereumAddressV2Field, HexField, Keccak256Field
+from gnosis.safe.safe_signature import SafeSignatureType
+
+logger = getLogger(__name__)
+
+
+class SafeMessage(TimeStampedModel):
+    """
+    Safe Message (EIP-191 or EIP-712) to build an EIP-1271 signature from
```

Comment:
nit: the message doesn't have to be an EIP-191, standards like [SIWE](https://eips.ethereum.org/EIPS/eip-4361) don't use EIP-191

Reply:
For EIP-4361 unless I'm missing something I see that they EIP191 for encoding the message